### DISCUSSION TITLE
Can be possible to have "music" when there is music and not speech in the stream?

### DISCUSSION THREAD

**Discussion by vtontodonato** [juanmc2005/diart#232] [General]
It can be usefull to have a special label "music" when there is music in the stream to better identify situation in speech stream analysis. Any Idea how to change the segmentation section to have such information?

**Comment by juanmc2005** [comment]
Hi @vtontodonato, the segmentation models are not able to detect music. If you want that feature you should use a model trained for music detection. This kind of detection/classification tasks are not currently supported by diart but they could easily be with a new block `AudioClassification` or something similar. Let me know if you want to contribute for this feature, I would welcome a PR and I can guide you through it if you find a suitable model :)

**Reply by vtontodonato** [reply]
Thanks, it will be interesting to manipulate the code to add such a feature. Unfortunatelly I cannot schedule this in the next weeks. But I'll try and I'll let you know my advance on such topic. Thanks.
